### DISCUSSION TITLE
Reports > Trends always Indeterminated

### DISCUSSION THREAD

**Discussion by hamdyaea** [ConSol/omd#185] [Q&A]
![Capture](https://github.com/ConSol-Monitoring/omd/assets/25694730/27c1ecb0-879c-40e8-8679-587f2105efc9)

The trends are always Indeterminated. I use Thruk as GUI and Naemon as core.

The version of omd labs is : 5.30

I need to know from which files is produced the Trends ?

Can I do something to repare it ?

**Comment by hamdyaea** [comment]
After fast one week where I can see the logs in Thruk and naemon is working correctly I still have no trends. (always Indeterminate 100% for all hosts and service.)

**Reply by sni** [reply]
moved this into the discussion section.
The naemon logs in omd are in ~/var/naemon/naemon.log and ~/var/naemon/archive

**Reply by hamdyaea** [reply]
Can you tell me if Trends is using only the nameon.log file or also the archives ?

**Reply by sni** [reply]
it's using both. Actually it's a bit more complicated. Thruk simply connects to naemon(-livestatus) and requests logs for a given timerange. But naemons livestatus uses the current logfile along with the archive.
